base-server/index.js

```diff
@@ -630,6 +643,10 @@ export class BaseServer {
     let key = this.lastClient.toString()
     let client = new ServerClient(this, connection, key)
     this.connected.set(key, client)
+    if (this.queueTypes.length) {
```

Comment:
Even if all actions uses `main` queue, actions need to be processed with a user’s queue. Why do we have here a `if`?

Reply:
I thought it might be beneficial to add this functionality in as non invasive way as possible, so if the developer decides that all potential actions are safe, queues can be neglected entirely as there are no actions that should be processed in queue and everything is presumably safe to process in parallel
That is also why I used array instead of `Map` for `queueTypes`. I can rework my solution to have a `main` queue.
Does it mean that actions in the `main` queue, should be processed in parallel, or should the queue be default way to process actions with ability to opt out for certain types?

Reply:
Yes, unfortunately, we can’t add a new feature, we need to refactor the old one.

Old logic (all tasks are processing in parallel) leads to painful issues (like `logux/subscribe` to post `A` was executed before `posts/create` A was not finished).

Reply:
I understand, I will rework my solution to support this